tests/test_asyncio.py

```diff
@@ -20,6 +20,15 @@
 import pytest
 
 
+@pytest.fixture
+def pytester(pytester: pytest.Pytester) -> pytest.Pytester:
```

Comment:
Inside the `pytester` fixture they create a new `pytest.Pytester` instance which inside their constructor overrides these env vars. This overrides the `pytester` fixture so it runs after it was created, resets the env vars and returns the original instance back.